src/theme/dark/colors.js

```diff
@@ -12,6 +12,7 @@ const appColors = {
   dropdown: rawColors.neutral.outerSpace,
   elementBackground: rawColors.neutral.outerSpace,
   elementBackgroundHover: rawColors.neutral.tuna,
+  headerItemBackground: rawColors.neutral.limedSpruce,
```

Comment:
Use one of the other colors that are blackhaze on the light theme.

Reply:
I have talked with the design and told me to use `limed spruce` in dark theme!
Do you need me to re-sync with them and ask them if an alternative color could be used here?

Reply:
That is my question thought...
There is no current color matching `rawColors.neutral.blackhaze` in default theme and `rawColors.neutral.limedSpruce`, so since I have already synced with the designer on which color to use for dark theme, if you are against of adding an extra color for this case (which I don't get why to be honest), would you like me to do an other sync with the design and ask if it's ok to use one of the existing ones for dark theme, instead of `rawColors.neutral.limedSpruce`?

Reply:
I never asked you to sync with the design team. You got a design which was either on light or on dark mode. You have `blackhaze` somewhere, so you use one of the existing colors that we are using across the app.
For example `elementBackground` which makes total sense. You don't ask anyone else in such case.
Now if we see a color that doesn't make sense with the existing colors, then we talk to the designer and we try to find a way to NOT add another color, just because someone told me on slack. We try to figure out if something of our design system lives up to expectations of the designer.

Reply:
Ok, thanks! I'll notify design that I will be using one of the existing pairs and remove the commit!